cpp/tests/hashing/hash_test.cpp

```diff
@@ -736,12 +736,99 @@ TEST_F(SparkMurmurHash3Test, StringsWithSeed)
   CUDF_TEST_EXPECT_COLUMNS_EQUAL(*hash_strings, hash_strings_expected_seed_314, verbosity);
 }
 
-TEST_F(SparkMurmurHash3Test, ListThrows)
+TEST_F(SparkMurmurHash3Test, ListValues)
 {
-  lists_column_wrapper<cudf::string_view> strings_list_col({{""}, {"abc"}, {"123"}});
-  EXPECT_THROW(
-    cudf::hash(cudf::table_view({strings_list_col}), cudf::hash_id::HASH_SPARK_MURMUR3, {}),
-    cudf::logic_error);
+  /*
+  import org.apache.spark.sql.functions._
```

Comment:
This is definitely kind of helpful, but I'm not sure that the whole comment really belongs here. We don't do that in any of our other tests. Our files will quickly bloat if we do include things like this. Maybe we need a better way to document our Spark-related tests?

Reply:
Reproducing the reference values for these tests is extremely difficult without code samples. I spent a considerable number of hours figuring out how to generate the reference values, and would not want anyone else to go through that. We could create separate files like `spark_list_hash.scala` and say `/* See spark_list_hash.scala to reproduce these values. */` but I don't like adding extra files for that. I think it's essential to include _something_ here for future reproduction. Would you propose an alternative?

Reply:
If it's that difficult I'm fine with leaving it there for now. If we find this issue recurring then yes, I would advocate for separate files (or at least one separate file containing all examples, for instance), but given that you think this is a significant value-add and it's not something we do for many other tests we can leave this here for now.